backend_python_scripts/text_analysis_functions.py

```diff
@@ -20,49 +17,32 @@ def scrape_frontpage_terms(page='http://www.politifact.com',stopwords_csv=None):
     Returns:
     ==========
     cleaned_words : all of the alphanumeric visable text found on the website
-
     """
 
-    stopwords = [] # OK if empty
-    if stopwords_csv:
-        with open(stopwords_csv, 'rb') as csvfile:
-            reader = csv.reader(csvfile,delimiter='\n')
-            for row in reader:
-                stopwords.append(row[0])
-
-    req = urllib2.Request(page)
-
-    response = urllib2.urlopen(req)
-    page_content = response.read()
-    soup = BeautifulSoup(page_content,'xml')
+    page = requests.get(url)
+    soup = BeautifulSoup(page.content, 'xml')
 
     # kill all script and style elements
     for script in soup(["script", "style"]):
         script.extract()    # rip it out
 
     # get text
     text = soup.get_text()
+    text_tokenize = word_tokenize(text.replace('\n', ' ').replace('-', ' '))
 
-    # break into lines and remove leading and trailing space on each
-    lines = (line.strip() for line in text.splitlines())
-    # break multi-headlines into a line each
-    chunks = (phrase.strip() for line in lines for phrase in line.split("  "))
-    # drop blank lines
-    text = [chunk.split() for chunk in chunks if chunk]
-
-    cleaned_words = []
-    for words in text:
-        for word in words:
-            word = word.lower()
-            word = filter(lambda x: x.isalnum(),word)
-            if word not in stopwords:
-                cleaned_words.append(word)
+    text_tokenize = word_tokenize(text_replace)
```

Comment:
text_replace isn't defined. I think this just got left over from the line above. It works fine if I comment this out.